src/hooks/useButtonAnimation.ts

```diff
@@ -25,8 +28,10 @@ const useButtonAnimation = (
     }
     return () => {
       isMounted = false
+      clearTimeout(timeout)
```

Comment:
Si on est pas rentré dans le if qui fait `timeout = ...` alors on va faire un `clearTimeout(undefined)` non ?

Reply:
According to the documentation on MDN 
[Notes](https://developer.mozilla.org/en-US/docs/Web/API/clearTimeout#notes)
Passing an invalid ID to clearTimeout() silently does nothing; no exception is thrown.